repos/system_upgrade/common/actors/initramfs/upgradeinitramfsgenerator/files/generate-initram.sh

```diff
@@ -67,6 +67,19 @@ build() {
         DRACUT_MODULES_ADD=$(echo "--add $LEAPP_ADD_DRACUT_MODULES" | sed 's/,/ --add /g')
     fi
 
+    case $LEAPP_DEVEL_INITRAM_NETWORK in
+        network-manager)
+            DRACUT_MODULES_ADD="$DRACUT_MODULES_ADD --add network-manager"
+            touch /etc/leapp-initram-network-manager
+        ;;
+        scripts)
+            DRACUT_MODULES_ADD="$DRACUT_MODULES_ADD --add network";
+            touch /etc/leapp-initram-network-scripts
+        ;;
+        *)
+        ;;
+    esac
```

Comment:
@fernflower this should be rather implemented via `UpgradeInitramfsTasks`  and `TargetUserSpaceUpgradeTasks` instead of adding another logic into the script.